### ISSUE TITLE
Clarification: Beginning of time series

### ISSUE THREAD

**Issue by Nora-Go** [acominola/STREaM#2]
Does the output time series begin on the first day at hour 0? Or another time?
Thank you for the clarification!

**Comment by acominola** [comment]
Yes, you are right. STREaM generates flow traces for whole days (24 hours).
